### DISCUSSION TITLE
Paclet repository

### DISCUSSION THREAD

**Discussion by woct0rdho** [FeynCalc/feyncalc#240] [General]
Is there any plan to submit FeynCalc to the newly announced [Wolfram Language Paclet Repository](https://resources.wolframcloud.com/PacletRepository/), so that it's easier to install the paclet and manage the namespace?

**Comment by vsht** [comment]
No. The WL Paclet repository imposes too many requirements that do not suit every package. Like we ditched the documentation
using notebooks for good are we are definitely not coming back to it.
